### PR TITLE
Add link to terminal apps

### PR DESCRIPTION
**Pull request by yanick**
I also squished the table into two Work/Don't Work sections.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,19 +56,23 @@ Swift:
...
-| **Terminal.app** | **GNOME Terminal** |
-| **Termux**       | **mate-terminal**  |
-| **upterm**       | **mintty**         |
-| **ZOC** (macOS)  | **PuTTY**          |
-| **iTerm 2** ([since 3.1](https://gitlab.com/gnachman/iterm2/issues/3568#note_13118332)) | **rxvt** |
-|                  | **ZOC** (Windows)  |
-|                  | **gtkterm, guake, LXTerminal, sakura, Terminator, xfce4-terminal,** and other libvte-based terminals ([bug report](https://bugzilla.gnome.org/show_bug.cgi?id=584160)) |
+### Works
+
+**[Butterfly][butterfly]**, **Hyper.app**, **Konsole**, **QTerminal**,
```
Comment (j-f1) [inline README.md]: I think you can just do `[Butterfly]` and the link will work.

`Hyper.app` can be found at https://hyper.is/.

**Comment by tonsky** [comment]
Thanks, but I’m fine with the table

**Comment by yanick** [comment]
Fair enough. :-) Do you want me to create another PR with just the addition of the links, or would you prefer I don't?

**Comment by tonsky** [comment]
No, thanks. I don’t think finding a particular terminal app is a problem these days

**Comment by yanick** [comment]
Oh, it's not hard at all, but I was thinking that since I was checking them anyway, I could make it a tad easier for the next person.

Oh, and btw, thanks for the awesome font. Ever since I installed it, each time I give a live demo somewhere, the question period *always* start with "yeah, yeah, thanks, but what is your font? It's AMAZING!"

**Comment by tonsky** [comment]
oh, thanks for the kind words! I’m glad you’re enjoying it

**Pull request closed without merging**